packages/core/stories/layout/grid-layout.doc.stories.mdx

```diff
@@ -32,7 +29,7 @@ Your content is best displayed using a configurable number of columns and rows.
 Grid layout can take any number of children, and will by default display them in a 12 columns grid (items after the 12th one will wrap).
 In order to change the number of columns displayed in the grid, you can reduce or increase the `columns` prop.
 
-Grid layout is useful when you want to control the layout in both row and columns dimensions and it can achieve more complex forms when used in combination with `GridItems`.
+Grid layout is useful when you want to control the layout in both row and columns dimensions, and it can achieve more complex forms when used in combination with `GridItems`.
```

Comment:
```suggestion
Grid layout is useful when you want to control the layout in both row and column dimensions, and it can achieve more complex forms when used in combination with `GridItems`.
```